tests/contrib/grpc/test_grpc.py

```diff
@@ -40,6 +51,64 @@ def setUp(self):
         Pin.override(constants.GRPC_PIN_MODULE_CLIENT, tracer=self.tracer)
         self._start_server()
 
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_IAST_ENABLED="1"))
```

Comment:
I'd move those test to a new file test_grpc_apsec_iast.py and I'd add it to `.github/CODEOWNERS`

Reply:
It would imply duplicating all the grpc testing scaffolding, not sure if worth it.

Reply:
Done, please take a look.